### PR TITLE
add test for bug 6617

### PR DESCRIPTION
**Pull request by inashivb**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/6617

### CODE REVIEW

File: tests/bug-6617/test.rules [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alert http any any -> any any (msg:"Filestore toserver"; filestore:to_server,flow; sid:1; rev:1;)
+alert http any any -> any any (msg:"Filestore toclient"; filestore:to_client,flow; sid:2; rev:1;)
```
Comment (catenacyber) [inline tests/bug-6617/test.rules:2]: The bug happens if we have only one of these, right ?

File: tests/bug-6617/test.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    count: 0
+    match:
+      alert.signature: Filestore toserver
+      alert.signature_id: 1
+
+- filter:
+    count: 1
+    match:
+      alert.signature: Filestore toclient
+      alert.signature_id: 2
```
Comment (catenacyber) [inline tests/bug-6617/test.yaml:17]: The test should check the stored files (so file info events with the stored property)

Comment (catenacyber) [reply tests/bug-6617/test.yaml:17]: More precisely, the bug is that files will be stored whatever their direction

Comment (inashivb) [reply tests/bug-6617/test.yaml:17]: hmmm I think I understood that but misunderstood `filestore: to_server,flow` keyword. I thought it'll only alert if it is able to store a file in that dir.

Comment (catenacyber) [reply tests/bug-6617/test.yaml:17]: If I understand correctly, `filestore` keyword is postmatch action

**Pull request closed without merging**